### PR TITLE
Initial draft of angular struts tapestry post

### PR DESCRIPTION
**Pull request by superchris**
May have more to add, but it's something to start

### CODE REVIEW

File: posts/2013-07-09-angular-struts-tapestry.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+templates also harkens back to Tapestry: you use plain old html decorated with
+elements and attributes that map to angular directives. My strong hope is that
+this will let us work with designers in the same way: the html/css we build
+initially doesn't have to be translated into some other template language by
+developers.
+
+The component approach of angular also has also resulted in a similarly
+ecosystem of reusable components. Though I've heard developers scoff at the
+idea of a component library being valuable, as a wise friend once remarked:
+"No code is faster than code". Though he was talking about performance, it
```
Comment (cdmwebs) [inline posts/2013-07-09-angular-struts-tapestry.md]: Did you accidentally a word?

Edit: I get it.

Comment (superchris) [inline posts/2013-07-09-angular-struts-tapestry.md]: I accidentally all over everything

**Comment by joelturnbull** [comment]
ready @superchris?

**Comment by cdmwebs** [comment]
Merged in to posts.

**Pull request closed without merging**